rootfs/api/models/release.py

```diff
@@ -136,36 +134,38 @@ def previous(self):
             prev_release = None
         return prev_release
 
-    def rollback(self, user, version):
-        if version < 1:
-            raise EnvironmentError('version cannot be below 0')
-
-        summary = "{} rolled back to v{}".format(user, version)
-        prev = self.app.release_set.get(version=version)
-        new_release = self.new(
-            user,
-            build=prev.build,
-            config=prev.config,
-            summary=summary,
-            source_version='v{}'.format(version)
-        )
-
+    def rollback(self, user, version=None):
         try:
+            # if no version is provided then grab version from object
+            version = (self.version - 1) if version is None else int(version)
```

Comment:
it's just more code creep that I'd rather keep separate, for example changing the function signature and how we retrieve the version number from that is not important to this PR. It just slows down code review because I have to do a context switch in my mind to review this. It looks fine to me though.